docs/tutorial/part-eight/index.md

```diff
@@ -1,94 +1,93 @@
 ---
-title: Preparing a Site to Go Live
+title: Preparing a Site to Go Live 
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Wow! You've come a long way! You've learned how to:
+वाह! आप बहुत दूर आ चुके है! आपने सीख लिया है की कैसे:
 
-- create new Gatsby sites
-- create pages and components
-- style components
-- add plugins to a site
-- source & transform data
-- use GraphQL to query data for pages
-- programmatically create pages from your data
+- एक नया Gatsby साइट बनाया जाए
+- पेजेज और कंपोनेंट्स बनाया जाए
+- कंपोनेंट्स को स्टाइल किया जाए 
+- साइट मे प्लगिन्स ऐड किया जाए 
+- डाटा को सोर्स और ट्रांसफॉर्म किया जाए
+- GraphQL द्वारा पेजेज के लिए डाटा को क्वेरी किया जाए 
+- प्रोग्राम द्वारा अपने डाटा के लिए पेजेज बनाया जाए
 
-In this final section, you're going to walk through some common steps for preparing a site to go live by introducing a powerful site diagnostic tool called [Lighthouse](https://developers.google.com/web/tools/lighthouse/). Along the way, we'll introduce a few more plugins you'll often want to use in your Gatsby sites.
+इस आखरी भाग में, आप साइट को तैयार करने के लिए कुछ सामान्य चरणों से गुजरने जा रहे हैं ,आपको एक शक्तिशाली साइट डायग्नोस्टिक टूल से इंट्रोड्यूस करवाया जायेगा जिसका नाम [Lighthouse](https://developers.google.com/web/tools/lighthouse/) है| रास्ते में हम आपको कुछ और प्लगिन्स के साथ परिचित करवाएंगे जिनका आप अपने Gatsby साइट में कई बार इस्तेमाल करना चाहेंगे|
```

Comment:
`|` => `।`

change this everywhere in the whole document, something like find and replace